ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-connectivity.j2

```diff
@@ -0,0 +1,76 @@
+{# eos - monitor connectivity configuration #}
+{% if monitor_connectivity is arista.avd.defined %}
+!
+monitor connectivity
+{%     for vrf in monitor_connectivity.vrfs | arista.avd.natural_sort('name') %}
+{%         if vrf.name is arista.avd.defined %}
+   vrf {{ vrf.name }}
+{%             for interface_set in vrf.interface_sets | arista.avd.natural_sort('name') %}
+{%                 if interface_set.name is arista.avd.defined and interface_set.interfaces is arista.avd.defined %}
+      interface set {{ interface_set.name }} {{ interface_set.interfaces }}
+{%                 endif %}
+{%             endfor %}
+{%             if vrf.local_interfaces is arista.avd.defined %}
+      local-interfaces {{ vrf.local_interfaces }} address-only default
+{%             endif %}
+{%             if vrf.description is arista.avd.defined %}
+      description
+      {{ vrf.description }}
+{%             endif %}
+{%             for host in vrf.hosts | arista.avd.natural_sort('name') %}
+{%                 if host.name is arista.avd.defined %}
+      !
+      host {{ host.name }}
+{%                     if host.description is arista.avd.defined %}
+         description
+         {{ host.description }}
+{%                     endif %}
+{%                     if host.local_interfaces is arista.avd.defined %}
+         local-interfaces {{ host.local_interfaces }} address-only
+{%                     endif %}
+{%                     if host.ip is arista.avd.defined %}
+         ip {{ host.ip }}
+{%                     endif %}
+{%                     if host.url is arista.avd.defined %}
+         url {{ host.url }}
+{%                     endif %}
+{%                 endif %}
+{%             endfor %}
+{%         endif %}
```

Comment:
we need an `exit` for leaving the VRF. I tested the molecule output, and it configured everything under the VRF.
Alternative is to swap the order, but then we don't match the eos order.
Another alternative is to always output either `shutdown` or `no shutdown` since these will go back to the main config mode.

Reply:
S, let's keep the vrf output in the end.